### DISCUSSION TITLE
Need update to newer version of Firefox, just like Thorium 126 updated.

### DISCUSSION THREAD

**Discussion by weslyraji** [Alex313031/Mercury#208] [General]
Hi, Thank you for update Thorium to 126, its a beast congrats for that, can we have similar update for Mercury as well ?

**Comment by rohitsaha29** [comment]
fingers crossed, and hopefully not ESR version and on main firefox. Alex will come through hopefully!

**Comment by MarcoRavich** [comment]
We do need GH actions's automated builds: realtime browser updates are crucial nowdays !

[How to Use Github Actions to Automate Your Repository Builds](https://www.howtogeek.com/devops/how-to-use-github-actions-to-automate-your-repository-builds/)

**Comment by gz83** [comment]
Updated to 129. In addition, Mercury does not plan to use GitHub Actions, which is not very helpful for rebasing and development, and may only have a certain impact on the compilation stage.
